packages/core/saved-objects/core-saved-objects-migration-server-internal/src/document_migrator/document_migrator.ts

```diff
@@ -6,55 +6,16 @@
  * Side Public License, v 1.
  */
 
-/*
- * This file contains logic for transforming / migrating a saved object document.
- *
- * At first, it may seem as if this could be a simple filter + reduce operation,
- * running the document through a linear set of transform functions until it is
```

Comment:
This long wall of text is mostly outdated, and doesn't really bring anything, so I decided to finally get rid of it.

Speak now or forever hold your peace.